codegen/lib/coin_test_gen.rb

```diff
@@ -61,7 +63,13 @@ def generate_coin_test_file(coin, templateFile, overwriteExisting = true)
     template = ERB.new(File.read(path), nil, '-')
     result = template.result(binding)
 
-    folder = 'tests/' + format_name(coin['name'])
+    folder = 'tests/'
+    if coin.key?('testFolderName')
+      folder += format_name(coin['folderName'])
+    else
+      folder += format_name(coin['name'])
+    end
+
```

Comment:
Existence of `testFolderName` key is checked, but then it is not used. Is it intended?

Reply:
oops, a typo